sycl/doc/SpecializationConstants.md

```diff
@@ -197,96 +197,110 @@ and the user says
 
 ##### The SpecConstant pass changes
```

Comment:
Why "changes"?
IMO the document should describe the design(probably with some implementation details) like it is _already done_, not like a _proposal_ to change the design or implementation.

Reply:
I guess it intended to meant "changes compared to scalars handling", but I agree with you, removed "changes" in 879566ac6f71c4a82d1bf2eb7933df584982b90c